src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -168,6 +172,16 @@ public void UpdateLocation()
             {
                 var positionMethod = typeof(Popup).GetMethod("UpdatePosition", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance);
                 positionMethod.Invoke(stepUIPopup, null);
+
+                if(stepUIPopup is PopupWindow)
+                {
+                    var stepUiPopupWindow = (PopupWindow)stepUIPopup;
+                    if (stepUiPopupWindow.webBrowserWindow != null && stepUiPopupWindow.webBrowserWindow.IsOpen)
+                    {
+                        var positionMethodWebBrowser = typeof(Popup).GetMethod("UpdatePosition", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance);
```

Comment:
I suggest that in order to not repeat this code that already exists in Step.UpdateLocation() method you can update the UpdateLocation method (probably can be static is up to you) and pass a parameter a Popup so we can execute the UpdatePosition in any popup and we don't need to repeat code (also I think you can include the null validation and IsOpen in the UpdateLocation method). Thanks